kaggle_environments/envs/werewolf/test_werewolf.py

```diff
@@ -0,0 +1,122 @@
+
+import pytest
+
+from kaggle_environments import make
+
+"""
+{
+  "roles": ["WEREWOLF", "VILLAGER", "VILLAGER", "SEER", "DOCTOR"],
+  "names": ["gpt-4o", "claude-3", "gemini-pro", "player-4", "player-5"],
+  "player_thumbnails": {
+    "gpt-4o": "https://images.seeklogo.com/logo-png/46/1/chatgpt-logo-png_seeklogo-465219.png",
+    "claude-3": "https://images.seeklogo.com/logo-png/55/1/claude-logo-png_seeklogo-554534.png",
+    "gemini-pro": "https://logos-world.net/wp-content/uploads/2025/01/Google-Gemini-Symbol.png"
+  }
+}
+"""
+
+
+URLS = {
+    "gemini": "https://logos-world.net/wp-content/uploads/2025/01/Google-Gemini-Symbol.png",
+    "openai": "https://images.seeklogo.com/logo-png/46/1/chatgpt-logo-png_seeklogo-465219.png",
+    "claude": "https://images.seeklogo.com/logo-png/55/1/claude-logo-png_seeklogo-554534.png",
+    "grok": "https://images.seeklogo.com/logo-png/61/1/grok-logo-png_seeklogo-613403.png"
+}
+
+
+@pytest.fixture
+def env():
+    roles = ["Werewolf", "Werewolf", "Doctor", "Seer", "Villager", "Villager", "Villager"]
+    names = [f"player_{i}" for i in range(len(roles))]
+    thumbnails = [URLS['gemini'], URLS['gemini'], URLS['openai'], URLS['openai'], URLS['openai'], URLS['claude'], URLS['grok']]
+    agents_config = [{"role": role, "id": name, "agent_id": "random", "thumbnail": url} for role, name, url in zip(roles, names, thumbnails)]
+    env = make(
+        'werewolf',
+        debug=True,
+        configuration={
+            "agents": agents_config
+        }
+    )
+    return env
+
+
+def test_load_env(env):
+    agents = ['random'] * 7
+    env.run(agents)
+
+    for i, state in enumerate(env.steps):
+        env.render_step_ind = i
+        out = env.renderer(state, env)
+        print(out)
+
+
+def test_discussion_protocol():
+    roles = ["Werewolf", "Werewolf", "Doctor", "Seer", "Villager", "Villager", "Villager"]
+    names = ["gemini-2.5-pro", "gemini-2.5-flash", "gpt-4.1", "o3", "o4-mini", "claude-4-sonnet", "grok-4"]
```

Comment:
I don't think we can run this test as you'll need the keys checked into the build pipeline (which we currently dont' have). Mark this as skip

Reply:
SG